test/acceptance/pages/select-business-page.js

```diff
@@ -116,6 +117,7 @@ class SelectBusinessPage extends CommonActions {
   async sheepReview() { await this.clickOn(SHEEP) }
   async beefCattleReview () { await this.clickOn(BEEF_CATTLE) }
   async dairyCattleReview () { await this.clickOn(DAIRY_CATTLE) }
+  async pigCattleReview () { await this.clickOn(PIG) }
```

Comment:
Suggest pigsReview would be a better name.
Also if you use PIGS instead of PIG then it's consistent with the others being plural - not a big deal but I think it helps to maintain the quality.